src/cli/README.md

```diff
@@ -746,22 +746,69 @@ Perform an MLE Discovery operation.
 Done
 ```
 
-### dns resolve \<hostname\> \[DNS server IP\] \[DNS server port\]
+### dns config
 
-Send DNS Query to obtain IPv6 address for given hostname. The latter two parameters have following default values:
+Get the default query config used by DNS client.
 
-- DNS server IP: 2001:4860:4860::8888 (Google DNS Server)
-- DNS server port: 53
+The config includes the server IPv6 address and port, response timeout in msec (wait time to rx response), maximum tx attempts before reporting failure, boolean flag to indicate whether the server can resolve the query recursively or not.
+
+```bash
+> dns config
+Server:[fd00:0:0:0:0:0:0:1]:1234
```

Comment:
We typically have space after the `:` in other CLI output. We should be consistent. Is there a preference to remove the space?

Reply:
I will change this, I also like the space after `:` when we list items per line (looks cleaner).